platform/view/services/db/driver/sql/sqlite/vault.go

```diff
@@ -51,18 +51,13 @@ func newTxCodePersistence(readDB *sql.DB, writeDB common.WriteDB, tables common.
 
 func (db *VaultPersistence) Store(ctx context.Context, txIDs []driver.TxID, writes driver.Writes, metaWrites driver.MetaWrites) error {
 	span := trace.SpanFromContext(ctx)
-	span.AddEvent("start_store")
-	defer span.AddEvent("end_store")
+	span.AddEvent("Start store")
```

Comment:
the rest of the code still uses the convention of separating words with `_` and keeping everything lowercase. Was this intentional?